pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -1276,6 +1280,7 @@ func writeNewCheckpointFile(
 	filename string,
 	descBuf []byte,
 ) error {
+	filename = strings.TrimPrefix(filename, "/")
```

Comment:
can we add a comment about why we're doing this? I'd also love a test if there was something that was failing before this.

Reply:
So the thing that was failing was userfile specific, since userfile is the only one where it doesn't have file system semantics. So it could happen for anything where you pass a file path. For example I only used BACKUP INTO when testing but thats different than BACKUP TO for how it formats the path, so thats why I never caught the bug.

So I guess I could easily just write a test case that does BACKUP TO and SHOW BACKUP on a userfile to see if it works in that case, but I'm not sure if that really addresses the root issue?